integration_tests/test_employment.py

```diff
@@ -1,50 +1,46 @@
 import pytest
 
-from integration_tests.conftest import TIME_STEPS_TO_TEST
 from vivarium_census_prl_synth_pop.constants import data_values
 
 # TODO: Broader test coverage
 
+
 # TODO stop skipping once MIC-3703 has been implemented
-@pytest.mark.parametrize("time_step", TIME_STEPS_TO_TEST)
 @pytest.mark.skip
```

Comment:
Not related to this PR, but why do we skip this test? We should probably add a `reason=`

Reply:
We skip because of the ticket in the comment above (MIC-3703)